### ISSUE TITLE
training hyperparamters for imagenet1000?

### ISSUE THREAD

**Issue by zyuh** [Yujun-Shi/CwD#6]
Hello, I would like to ask what is the hyperparameter setting of training Imagenet1000, is it the same as that of Imagenet100? Cou ld you provide shell scripts?

**Comment by Yujun-Shi** [comment]
hello @zyuh, thanks for your interests in our work!
For all the other basic hyper-parameters, they are the same.
The only difference is that we set the coefficient for our CwD objective to be **1.5** for ImageNet1000 experiments.

For the current version of code, we cannot run imagenet1000. This is because we found that the step of finetuning classifier after learning each task is very important (about 3% different in final accuracy) and we don't have the proper implementation in this repo.

We run all our imagenet1000 experiments w/ LUCIR and PODNet by adding CwD code on top of the PODNet repo (https://github.com/arthurdouillard/incremental_learning.pytorch).

If you're in a hurry, you can try add our CwD code on top of the PODNet repo and run the experiments, or you can wait for our released code with imagenet1000.

**Comment by zyuh** [comment]
hello @Yujun-Shi 
thanks for your reply！
I want to know whether the PODNet repo contains the "the step of finetuning classifier" you said？
And the procedures for "finetuning classifier after learning each task" the same as those for "balanced ﬁnetuning" in EEIL[1] or BIC[2]?

[1]  End-to-End Incremental Learning
[2]  Large Scale Incremental Learning

**Comment by Yujun-Shi** [comment]
@zyuh Yes, it contains the classifier finetuning. It's somewhat like BiC but it's not exactly the same. It was the balance finetuning introduced in the second paragraph of Sec. 3.5 in LUCIR paper (https://openaccess.thecvf.com/content_CVPR_2019/papers/Hou_Learning_a_Unified_Classifier_Incrementally_via_Rebalancing_CVPR_2019_paper.pdf). Basically, it's a component of LUCIR.
